backend/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Essa linha pode ter o script de `start`
```suggestion
    "start": "node ."
```